packages/core/workers/src/Profiler.js

```diff
@@ -39,17 +39,22 @@ type PositionTickInfo = {|
 export default class Profiler {
   session: Session;
 
-  startProfiling() {
-    this.session = new Session();
-    this.session.connect();
+  async startProfiling() {
+    try {
+      const inspector: any = await require('inspector');
```

Comment:
`require` is synchronous. No need to `await`

Reply:
Also, `inspector` is still being imported at the top of the file. Was that intentional?

Reply:
Yes exactly, but we will have type issue in this line: 

https://github.com/parcel-bundler/parcel/blob/42f3276c86a8da0a8cb1a1f462e8589eb82ff68e/packages/core/workers/src/Profiler.js#L40

Reply:
Use `import type {Session} from ...` or `import typeof {Session} from ...`

Reply:
Thanks, this is what I was looking for!